src/components/pill/styled.js

```diff
@@ -1,12 +1,27 @@
 import styled from "styled-components"
 import Flex from "src/components/templates/flex"
-import getPillHeight from "src/components/pill/mixins/height";
+import getPillBackground from "./mixins/background"
+import { getPillColor } from "./mixins/colors"
+import getPillHeight from "./mixins/height"
+import getPillPadding from "./mixins/padding"
+import getPillWidth from "./mixins/width"
 
-export const MasterCardContainer = styled(Flex).attrs(
+export const AlertsContainer = styled(Flex).attrs(
   ({ background, height, onClick, round = 999, size }) => ({
     background,
+    ...(onClick && { cursor: "pointer" }),
+    height: getPillHeight(height, size),
+    round,
+    size,
+  })
+)``
+
+export const MasterCardContainer = styled(Flex).attrs(
+  ({ background, height, isAlert, onClick, round = 999, size }) => ({
+    background,
+    ...(onClick && { cursor: "pointer" }),
     height: getPillHeight(height, size),
-    cursor: onClick ? "pointer" : "default",
+    margin: isAlert && [0, 0, 0, -1],
```

Comment:
Same, this coupling will make it difficult to maintain. You should enhance the simple styles, specifically for the alert in its own component. Or create flavours.